website/source/docs/configuration/providers.html.md

```diff
@@ -117,3 +117,14 @@ KEY {
 	CONFIG
 }
 ```
+
+## Interpolation
+Providers support [interpolation syntax](/docs/configuration/interpolation.html) allowing dynamic configuration at run time.
+
+```
+provider "aws" {
+	region = "${var.aws_region}"
+}
+```
+
+Only [variables](/docs/configuration/variables) and [remote state](/docs/state/remote.html) are supported, it is not possible to use the output from a resource, module or data source in the interpolation syntax for a provider.
```

Comment:
Thanks, I was just browsing some of the other docs I think this should be a note also.  

I was trying to make a note that you can't at the moment use outputs in the syntax, I was not really sure how to write this I will do another update after lunch.